mosip-identity-plugin/src/main/java/io/mosip/signup/plugin/mosipid/service/IdrepoProfileRegistryPluginImpl.java

```diff
@@ -209,8 +209,14 @@ public ProfileCreateUpdateStatus getProfileCreateUpdateStatus(String requestId)
     @Override
     public ProfileDto getProfile(String individualId) throws ProfileException {
         try {
-            String endpoint = String.format(getIdentityEndpoint, individualId);
-            ResponseWrapper<IdentityResponse> responseWrapper = request(endpoint, HttpMethod.GET, null,
+            IdRequestByIdDTO requestByIdDTO = new IdRequestByIdDTO();
+            RequestWrapper<IdRequestByIdDTO> idDTORequestWrapper=new RequestWrapper<>();
+            requestByIdDTO.setId(individualId);
```

Comment:
postfix is missing, we should take the postfix from the config and then append only if its exists. @pvsaidurga